src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -194,12 +195,16 @@ public System.Collections.Generic.IList<ITestResult> Children
         /// <summary>
         /// Gets a TextWriter, which will write output to be included in the result.
         /// </summary>
-        public StringWriter OutWriter
+        public TextWriter OutWriter
         {
             get
             {
                 if (_outWriter == null)
```

Comment:
Btw, the lazy initialization is not threadsafe. So if you are very unluckly, you can end up with two separate StringWriter instances writing to the same _output (each of them is synchronized on its own), which is still not threadsafe.